hcl/eval/object.go

```diff
@@ -205,7 +210,7 @@ func (obj *Object) SetAt(path ObjectPath, value Value) error {
 		// where's the definition of each value and overwrite the values if they
 		// come from a child scope or ignore new values that comes from the parent
 		// but conflicts with definitions in the child.
-		if !strings.HasPrefix(value.ConfigOrigin().String(), old.ConfigOrigin().String()) {
+		if !strings.HasPrefix(value.Origin().ConfigFrom.String(), old.Origin().ConfigFrom.String()) {
```

Comment:
hmm what did confuse me a little is that ConfigFrom is a dir and then DefinedAt is a file. but I get it now =P